openproblems/tasks/dimensional_reduction/metrics/root_mean_square_error.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import scipy as sp
+from scipy import spatial
+import pandas as pd
+from ....tools.decorators import metric
+
+
+@metric(metric_name="compare_rmse", maximize=True)
+def compare_RMSE(adata, method):
```

Comment:
naming convention isn't set yet but I'd rather this called `rmse(adata)`, and it must be applied the DR data in a generic location (as discussed in the API section.)

Reply:
I agree - I need to do some reformatting of this metric file, will make consistent will all of your suggestions, thanks.